performance/benchmarks/bm_python_startup.py

```diff
@@ -8,6 +8,33 @@
 from six.moves import xrange
 
 
+# common stdlibs. Many applications imports them directly, or indirectly.
```

Comment:
I'm not sure how @methane did it, but the public GitHub data set on Big Query can be a useful proxy for module popularity: https://cloud.google.com/bigquery/public-data/github

While it misses a lot of code (both proprietary and open source), it's still likely to be more representative than our own intuition.